catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -465,6 +477,23 @@ private void skipBytes(InputStream is, String bytesToSkip) throws IOException {
         }
     }
 
+    private void alignStream(InputStream in, long requestedBytesToSkip, long responseBytesSkipped)
+            throws IOException {
+        try {
+            if (requestedBytesToSkip > responseBytesSkipped) {
+                LOGGER.error(
```

Comment:
As this is an expected case as you said in your earlier comment and you're able to handle it by skipping N more bytes in the stream, I think this should be a WARN log.